README.md

```diff
@@ -1 +1,583 @@
+
+![Ballerina Observability Sample](/images/ballerina-observability-sample.png)
+
+## Prerequisites
+ 
+- [Ballerina Distribution](https://ballerina.io/learn/getting-started/)
+- A Text Editor or an IDE
+- [Docker](https://docs.docker.com/install/linux/docker-ce/ubuntu/)
+- [Prometheus](https://ballerina.io/learn/how-to-observe-ballerina-code/#prometheus)
+- [Grafana](https://ballerina.io/learn/how-to-observe-ballerina-code/#grafana)
+- [Jaeger](https://ballerina.io/learn/how-to-observe-ballerina-code/#jaeger-server)
+
+### Optional requirements
+- Ballerina IDE plugins ([IntelliJ IDEA](https://plugins.jetbrains.com/plugin/9520-ballerina), [VSCode](https://marketplace.visualstudio.com/items?itemName=WSO2.Ballerina), [Atom](https://atom.io/packages/language-ballerina))
+- [Kubernetes](https://kubernetes.io/docs/setup/)
+
+## Testing the services
+
+### Setting up the environment  
+
+Please refer to the [Prerequisites](#prerequisites) section for information on setting up the required tools and environment. Access the below URLs and make sure you have Prometheus, Grafana and Jaeger servers up and running.
+- Prometheus - [http://localhost:19090](http://localhost:19090)
+- Grafana - [http://localhost:3000](http://localhost:3000)
+- Jaeger - [http://localhost:16686](http://localhost:16686)
+
+Create the database using the script provided in resources directory. This datbase is being used by the services to retrieve data.
+```sql
+mysql -u <username> -p
+source resources/mysql.sql;
+```
+
+### Invoking the service
+
+- Navigate to `microservices-observability/guide` and run the following commands in separate terminals to start all four HTTP services. This will start the `Airline Reservation`, `Hotel Reservation`, `Car Rental` and `Travel Agency` services in ports 9091, 9092, 9093 and 9090 respectively.
+
+```bash
+   $ ballerina run travel_agency/
+```
+```bash 
+   $ ballerina run -e b7a.observability.metrics.prometheus.port=9797 airline_reservation/
+```
+```bash
+   $ ballerina run -e b7a.observability.metrics.prometheus.port=9798 hotel_reservation/
+```
+```bash
+   $ ballerina run -e b7a.observability.metrics.prometheus.port=9799 car_rental/
+```
+Note that we are overridng the prometheus port by passing the parameter (`-e b7a.observability.metrics.prometheus.port=xxxx`) for all the services, except the travel_agency service. Travel agency service will publish the metrics data on the default port defined in the `ballerina.conf` file (9796). This will avoid any conflicts on port being already occupied.
```

Comment:
prometheus -> Prometheus

Remove the parenthesis around the config param

Instead of referring to the travel agency service by its package name, shall we use the actual name of the service or maybe Travel Agency service? Whichever is used, need to use it consistently. Immediately, in the second sentence it says "Travel agency service".

Reply:
data on the -> data to the